test/unit/presenters/publishing_api/html_attachment_presenter_test.rb

```diff
@@ -93,4 +93,15 @@ def present(record)
 
     assert_equal %w[abcdef], present(html_attachment).links[:organisations]
   end
+
+  test "HtmlAttachment presents primary_publishing_organisation" do
+    create(:publication, :with_html_attachment, :published)
+
+    html_attachment = HtmlAttachment.last
```

Comment:
Would this work?

```ruby
html_attachment = create(:publication, :with_html_attachment, :published)
```

It would be more concise.

Reply:
Wouldn't that return the publication, not the attachment?

Reply:
Ah, yeah, good point.